tools/mo/requirements.txt

```diff
@@ -4,7 +4,7 @@ mxnet~=1.2.0; sys_platform == 'win32'
 mxnet~=1.7.0.post2; sys_platform != 'win32'
 networkx~=2.5; python_version <= "3.6"
 networkx<2.8.1; python_version > "3.6"
-protobuf>=3.15.6
+protobuf~=3.18.1
```

Comment:
Guys, I am concerned about such strict requirements. Please fix what is needed to relax upper bound

Reply:
I set up this strict requirement because we use protobuf 3.18 as a submodule and the same version was set up in the other places in #12275

Reply:
Artur, could you please clarify objective to set-up this strict version and can we relax this version on other places instead please?

Reply:
I restricted protobuf to version < 4.0.0 and to be sure that Python and ONNX are supported I increased the minimal version to 3.18.1

Reply:
Don't we need to update this in other places of OpenVINO like Model Zoo, NNCF?

Reply:
If Model Zoo and NNCF use protobuf in version 3.x.x then we don't have any risks that it will be not work